src/state/contexts/APIContext.js

```diff
@@ -155,19 +158,28 @@ const APIProvider = ({ children }) => {
   const getCustomerByID = authState => {
     const headers = getAuthHeader(authState);
 
-    return axios
-      .get(`${process.env.REACT_APP_API_URI}/customers/${userInfo.sub}`, {
-        headers,
-      })
-      .then(res => {
-        if (res.data) {
-          setCustInfo(res.data);
-          setIsRegistered(true);
-        }
-      })
-      .catch(err => {
-        console.log(err);
-      });
+    return (
+      axios
+        .get(`${process.env.REACT_APP_API_URI}/customers/${userInfo.sub}`, {
+          headers,
+        })
+        .then(console.log('userInfo.sub: ', userInfo.sub)) // this seems to be what's getting lost and crashing the customer dashboard
+        .then(res => {
+          if (res.data) {
+            setCustInfo(res.data);
+            setIsRegistered(true);
+          }
+        })
+        // .get(`${process.env.REACT_APP_API_URI}/customers/${userInfo.sub}`, {
+        //   headers,
+        // })
+        // .then(res => {
+        //   console.log('getCustomerByID: ', res)
```

Comment:
just make sure if you keep it to remove the console log

Reply:
This part of the test API call was removed as well, since I think that portion of the dashboard crash source has been traced.